src/system/Code/Code.tsx

```diff
@@ -46,6 +53,21 @@ const Code = React.forwardRef(
 			return codeDom;
 		}
 
+		const onClickCopy = () => {
+			const copyText = async () => {
+				await window.navigator.clipboard.writeText( ref.current?.innerText ?? '' );
+
+				setCopied( true );
+
+				if ( onCopy ) {
+					onCopy();
+				}
+			};
+
+			// eslint-disable-next-line @typescript-eslint/no-floating-promises
+			copyText();
```

Comment:
this is what i was trying to say before. you don't have to use async/await if you use then/catch properly. all of the lint errors are trying to get you to recognize that you are not properly error-handling

```suggestion
			window.navigator.clipboard
				.writeText( ref.current?.innerText ?? '' )
				.then( () => {
					setCopied( true );

					if ( onCopy ) {
						onCopy();
					}
				} )
				.catch( () => {} );
```